tools/pip/doc/CU101_ADDITIONAL.md

```diff
@@ -21,7 +21,7 @@ This package supports Linux and Windows platforms. You may also want to check:
 - [mxnet-cu112](https://pypi.python.org/pypi/mxnet-cu112/) with CUDA-11.2 support.
 - [mxnet-cu110](https://pypi.python.org/pypi/mxnet-cu110/) with CUDA-11.0 support.
 - [mxnet-cu102](https://pypi.python.org/pypi/mxnet-cu102/) with CUDA-10.2 support.
-- [mxnet-cu100](https://pypi.python.org/pypi/mxnet-cu101/) with CUDA-10.0 support.
```

Comment:
do we want to list the mxnet-cu101 variant too?

Reply:
Seems this part is a reference to other variants not including itself. Other files like `tools/pip/doc/CU100_ADDITIONAL.md` and `tools/pip/doc/CU102_ADDITIONAL.md` do not include the self link either

Reply:
got it, makes sense